README.md

```diff
@@ -32,19 +32,21 @@ A package that allows you to use the native file explorer to pick single or mult
 * Supports retrieving as XFile (cross_file) for easy manipulation with other libraries
 * Different default type filtering (media, image, video, audio or any)
 * Picking directories
+* Picking both files and directories simultaneously
 * Load file data immediately into memory (`Uint8List`) if needed; 
 * Open a save-file / save-as dialog (a dialog that lets the user specify the drive, directory, and name of a file to save)
 
 If you have any feature that you want to see in this package, please feel free to issue a suggestion. 🎉
 
 ## Compatibility Chart
 
-| API                   | Android            | iOS                | Linux              | macOS              | Windows            | Web                |
-|-----------------------|--------------------|--------------------|--------------------|--------------------|--------------------|--------------------|
-| clearTemporaryFiles() | :heavy_check_mark: | :heavy_check_mark: | :x:                | :x:                | :x:                | :x:                |
-| getDirectoryPath()    | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :x:                |
-| pickFiles()           | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
-| saveFile()            | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| API                           | Android            | iOS                | Linux              | macOS              | Windows            | Web                |
+|-------------------------------|--------------------|--------------------|--------------------|--------------------|--------------------|--------------------|
+| `clearTemporaryFiles()`       | :white_check_mark: | :white_check_mark: | :x:                | :x:                | :x:                | :x:                |
+| `getDirectoryPath()`          | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :x:                |
+| `pickFileAndDirectoryPaths()` | :x:                | :x:                | :x:                | :white_check_mark: | :x:                | :x:                |
```

Comment:
@philenius Out of curiosity, is there a possibility to add this to other platforms in the future?
Or is this stricly MacOS only?

Reply:
Well, then it's fine to have the API implemented as-is. _If_ there is a future possibility for other platforms, that can be addressed later.